litellm/llms/vertex_ai.py

```diff
@@ -532,9 +539,43 @@ def completion(
             """
             Vertex AI Model Garden
             """
+            ## LOGGING
```

Comment:
any reason for the changes to this branch ? I thought the pr was for `mode=private`

Reply:
I have covered this during our conversation last week -- I added info in MODELSthe request string and updated it to make `llm_model` have a more similar usage to it's definitions with the other non-custom models